official/resnet/keras/keras_cifar_main.py

```diff
@@ -32,27 +32,18 @@
 from official.utils.flags import core as flags_core
 from official.utils.logs import logger
 from official.utils.misc import distribution_utils
-from tensorflow.python.keras.optimizer_v2 import gradient_descent as gradient_descent_v2
 
 
-# LR_SCHEDULE = [    # (multiplier, epoch to start) tuples
-#     (1.0, 5), (0.1, 30), (0.01, 60), (0.001, 80)
-# ]
 LR_SCHEDULE = [  # (multiplier, epoch to start) tuples
     (0.1, 91), (0.01, 136), (0.001, 182)
 ]
 
-BASE_LEARNING_RATE = 0.1
 
 def learning_rate_schedule(current_epoch, current_batch, batches_per_epoch, batch_size):
   """Handles linear scaling rule, gradual warmup, and LR decay.
 
-  The learning rate starts at 0, then it increases linearly per step.
-  After 5 epochs we reach the base learning rate (scaled to account
-    for batch size).
-  After 30, 60 and 80 epochs the learning rate is divided by 10.
-  After 90 epochs training stops and the LR is set to 0. This ensures
-    that we train for exactly 90 epochs for reproducibility.
+  The learning rate starts at base learning_rate, then after 91, 136 and
```

Comment:
I do not really know why we give it such a schedule. I was under the impression that this is parameter that @tfboyd tuned for performance reasons?
@tfboyd wanna comment here?

Reply:
sorry i didn't mean you should explain the schedule. I just meant that instead of specifying the exact epochs here just say something generic like
"Scale learning rate at epoch boundaries provided in LR_SCHEDULE by the provided scaling factor"